sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -244,8 +243,10 @@ endpoint="https://<region>.api.cognitive.microsoft.com/"
 text_analytics_client = TextAnalyticsClient(endpoint, credential)
 
 documents = [
-    "Microsoft was founded by Bill Gates and Paul Allen.",
-    "Redmond is a city in King County, Washington, United States, located 15 miles east of Seattle.",
+    """
+    Microsoft was founded by Bill Gates and Paul Allen. Its headquarters are located in Redmond. Redmond is a
+    city in King County, Washington, United States, located 15 miles east of Seattle.
+    """,
```

Comment:
nit: the """ """ comment adds newlines to output

Reply:
Yeah, sorry. I added that to make it more clear this is a document visually. There are no print statements though, so I'm going to ignore your nit for now